src/snowflake/snowpark/_internal/ast_utils.py

```diff
@@ -722,3 +728,25 @@ def fill_sp_write_file(
             t = expr.copy_options.add()
             t._1 = k
             build_expr_from_python_val(t._2, v)
+
+
+def build_proto_from_callable(
+    expr_builder: proto.SpCallable, func: Callable, ast_batch: Optional[AstBatch] = None
+):
+    """Registers a python callable (i.e., a function or lambda) to the AstBatch and encodes it as SpCallable protobuf."""
+
+    expr_builder.name = "test"
```

Comment:
docstring doesn't match code here

Reply:
good catch! Yes, this function is actually being filled in as part of the UDF ticket [SNOW-1514712](https://snowflakecomputing.atlassian.net/browse/SNOW-1514712). I added a TODO and removed the code from line 738.

Reply:
(It's not relevant for the test)